go-controller/pkg/kubevirt/dhcp.go

```diff
@@ -88,14 +140,12 @@ func retrieveDNSServiceClusterIPs(k8scli *factory.WatchFactory) (string, string,
 	return clusterIPv4, clusterIPv6, nil
 }
 
-func ComposeDHCPv4Options(cidr, dnsServer, controllerName string, vmKey ktypes.NamespacedName) *nbdb.DHCPOptions {
+func ComposeDHCPv4Options(cidr, controllerName string, vmKey ktypes.NamespacedName) *nbdb.DHCPOptions {
 	serverMAC := util.IPAddrToHWAddr(net.ParseIP(ARPProxyIPv4)).String()
 	dhcpOptions := &nbdb.DHCPOptions{
 		Cidr: cidr,
 		Options: map[string]string{
 			"lease_time": fmt.Sprintf("%d", dhcpLeaseTime),
```

Comment:
not entirely related to this PR, but is there any reason for us to make the lease expire every hour in OVN-Kubernetes for VM Pods? I would assume the podIP, gateway, dnsserver all stay the same for the life of the VM pod no?

Reply:
The option is mandatory, so for hypershift we configured it with an hour, no special reason though, I prefer not to change this if we don't have customer's complains yet.